edelweiss_data/api.py

```diff
@@ -743,6 +744,7 @@ def decode(cls, d, api):
             created=iso8601.parse_date(d['created']),
             description=d['description'] if 'description' in d else None,
             metadata=d['metadata'] if 'metadata' in d else None,
+            row_count=d['rowcount'] if 'rowcount' in d else None,
```

Comment:
Unless we change the JSON field to `rowCount`, we should change the variable name to `rowcount`to be consistent.